share/cover-thumbnailer-gui.glade

```diff
@@ -29,19 +30,18 @@
     <property name="preview_widget_active">False</property>
     <signal name="delete-event" handler="on_filechooserdialog_delete_event" swapped="no"/>
     <child internal-child="vbox">
-      <object class="GtkBox" id="dialog-vbox3">
```

Comment:
A lot of this diff chatter is caused by me using a different version of the glade editor from what you used. I left all such changes in their own commit so they can be easily distinguished from changes related to new functionality.

This is the most recent package I could find for a GTK2-capable Glade that can be installed side-by-side with Glade 3.22.
http://old-releases.ubuntu.com/ubuntu/pool/universe/g/glade-3/glade-gtk2_3.8.5-0ubuntu1_amd64.deb
With an old libgladeui-1 package also scavenged from past Ubuntu releases, I was able to install and use this on Debian 10 with no problem.